config.json

```diff
@@ -22,7 +22,7 @@
     "solution": ["Sources/%{pascal_slug}/%{pascal_slug}.swift"],
     "test": ["Tests/%{pascal_slug}Tests/%{pascal_slug}Tests.swift"],
     "example": ["Sources/%{pascal_slug}/%{pascal_slug}Example.swift"],
```

Comment:
I think this also needed to be updated to `[".meta/Sources/%{pascal_slug}.swift"]` Probably forgotten in a previous PR.

Reply:
Looks like the example files are in `[".meta/Sources/%{pascal_slug}/%{pascal_slug}Example.swift"]`, so I updated the config to use that as default.